src/app/shared/components/main-nav/main-nav.component.html

```diff
@@ -0,0 +1,25 @@
+<mat-sidenav-container class="sidenav-container">
+  <mat-sidenav #drawer class="sidenav" fixedInViewport="false"
+      [autoFocus]="false"
+      [attr.role]="'dialog'"
+      [mode]="'over'"
+      [opened]="false">
+    <div class="nav-wrapper">
+      <mat-toolbar class="nav-head">
+        <button type="button" class="close" mat-icon-button (click)="drawer.toggle()">
+          <mat-icon class="close-icon">close</mat-icon>
+        </button>
+      </mat-toolbar>
+      <mat-nav-list class="nav-list">
+        <a *ngFor="let link of links" class="nav-link" mat-list-item [routerLink]="link.route" [routerLinkActive]="['active']" (click)="drawer.toggle()">{{link.name}}</a>
```

Comment:
as I can see we should have s separator between user links and admin links. Do we have one?